knees4bees/dream-up-pup
# Feature/home page

### What does this PR do?
Creates landing page

### Are there any known bugs?
Nope

### Summary of the changes
- Fetches and formats breed data from API
- Creates mock data file to use during development
- Creates landing page with dropdown menu for breeds and submit button

### Notes for the reviewer
There is next to no styling here; that is being saved for a future iteration.

### How should this be tested?
View the app in the browser and confirm there are no errors in the console, that the landing page is being displayed, that you can select a breed from the dropdown, and that the URL changes when you click the "Write story" button.
